x/posts/keeper/msg_server_test.go

```diff
@@ -669,7 +669,7 @@ func (suite *KeeperTestSuite) TestMsgServer_EditPost() {
 					types.REPLY_SETTING_EVERYONE,
 					time.Date(2020, 1, 1, 12, 00, 00, 000, time.UTC),
 					nil,
-					"cosmos1r9jamre0x0qqy562rhhckt6sryztwhnvhafyz4",
+					"cosmos13t6y2nnugtshwuy0zkrq287a95lyy8vzleaxmd",
```

Comment:
We should ensure the the owner of post stored is the same as the user without `EditOwnContent` permission in this test